src/terminal/adapter/ut_adapter/Adapter.UnitTests.vcxproj

```diff
@@ -28,6 +28,9 @@
     <ProjectReference Include="..\..\..\interactivity\base\lib\InteractivityBase.vcxproj">
       <Project>{06ec74cb-9a12-429c-b551-8562ec964846}</Project>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\renderer\base\lib\base.vcxproj">

```

Comment:
Taking a mental note to adjust the link line in the `sources` file to include ConRenderBase (thanks for calling this one to my attention!)

Reply:
I've done the merge and the other updates. Would you like me to do this too or should I leave it for you to fix? I'm not exactly sure what is required, but I thought maybe I just had to add back this line:

    $(WINCORE_OBJ_PATH)\console\open\src\renderer\base\lib\$(O)\ConRenderBase.lib \

Or is there more to it than that?

Reply:
There's a nonzero chance there's more to it than that (Base might depend on Interactivity, which would bring back in the whole kit&kaboodle). I'm happy to handle this one, but thank you!

Reply:
(The best way to know is to try and build it, and the best way to do that is _probably having an actual Windows enlistment_, and.. well, you know.)